Slider/Assets/Scripts/Sliders/Grid/STile.cs

```diff
@@ -236,4 +236,14 @@ protected void SetTileMapPositions(Vector3 pos)
 
         allTileMaps.transform.position = pos;
     }
+
+    public static bool AreAllTilesActive(List<STile> tiles)
```

Comment:
this seems like it should be in sgrid?

Reply:
I was thinking that because its checking the list of tiles and not all the tiles in the grid that's why it belonged in STile (see use in `AllBuoy` in `OceanGrid`) but i can move it

Reply:
yeah maybe rename it to `AreTilesActive` and move then, i feel like generally sgrid is responsible/conceptually responsible for managing stiles